practices/Git/commit_signing.md

```diff
@@ -0,0 +1,32 @@
+## Signing GIT commits with GPG key
+
+### Why sign commits?
+When we push commits to Git without signing them, the only source of truth who pushed the commit is the user-name and the email address. But anyone can easily update both of these fields for the existing commits, or push commits with fake details.
+
+Commit signing helps you verify that commits are actually coming from a trusted source. If you are working on sensitive code or in the open source world, it's crucial to verify commiter's signature and also not allowing anyone to modify your commits.
+
+### How to sign commits?
+Follow [this](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification) guide for setting up the GPG key and managing commit signature verification.
```

Comment:
Maybe link to GitLab and BitBucket too? Or just mention "these are steps for GitHub, but you can find similar instructions for other services."

Reply:
Done